### DISCUSSION TITLE
Interpolate Pixels in SLEAP GUI

### DISCUSSION THREAD

**Discussion by roomrys** [talmolab/sleap#1471] [Ideas]
A user had asked about decreasing video quality in SLEAP; however, SLEAP only displays an up-sampled (and therefore seemingly pixelated) version of the video. __The user stated that this pixelated version of the video was more difficult to label__... and I agree with them. Looking at things from far away make more sense to my mind... what if we gave users the option to interpolate the pixels and "fill in the blanks".

Sub discussion in #1459
---

>When I import videos into sleap to label I noticed that the quality is worse than playing them outside of the software. Our resolution is 1024x768 and frame rate is 24.99 (mp4). Do you have any idea why that might be or how I can improve that without increasing the resolution since I have reached my labs maximum? Thanks so much for helping me get started!
_Originally posted by @IsabelleSajonia in https://github.com/talmolab/sleap/discussions/1459#discussioncomment-6825012_

>In the Videos table there should be column for height and width of the video. Do those match your original video resolution? Or are you referring to a change in frame rate? SLEAP should not adjust the video (other than the option for color/grayscale). 
If the observation is qualitative, could it be because you are noticing an up-sampling of the video to fit to the window (and allow more precise labeling)?
_Originally posted by @roomrys in https://github.com/talmolab/sleap/discussions/1459#discussioncomment-6825244_

>Yes the height and width in the table match my original video so I think it is just qualitative. To me this looks harder to label (more pixelated).
_Originally posted by @IsabelleSajonia in https://github.com/talmolab/sleap/discussions/1459#discussioncomment-6825380_

**Comment by gitttt-1234** [comment]
Thanks for the suggestion @roomrys  @IsabelleSajonia! Closing this discussion, added an option to view the actual video in the "View" menu on top of the GUI! PR: https://github.com/talmolab/sleap/pull/2708

**Reply by jtgrodeo** [reply]
Hello! I just wanted to add to this, the Actual Size (1:1) seems to remove the upsampling, but only scales to the size of the window meaning to preserve the pixels of small section of the video I have to make the window equally small. 

Is there a way to have the UI preserve the pixel size no matter the size of the window when zooming in?
